app/templates/views/broadcast/preview-areas.html

```diff
@@ -27,22 +27,25 @@
 
   {% for area in broadcast_message.areas %}
     {% if loop.first %}
-      <ul class="area-list">
+      <div>
+        <p class="govuk-!-margin-right-2 area-list">Area: </p>
+        <ul class="area-list">
     {% endif %}
```

Comment:
I know we never have any scenarios in which an area is not provided - but the reason behind this is to only create an <ul> if there are objects in the array. Otherwise, we'd have an empty array, and a <p> tag that says 'Area: ' without any areas. If there ever were, hypothetically, a scenario in which no areas were provided, it would make more sense to have the if statement, than to remove it, from an accessibility standpoint

Reply:
To be fair, this was something in the code previously and I had just assumed that that was the rationale behind it - happy to remove if you still don't think it makes sense

Reply:
You could wrap the whole thing in an if if you want that:

```
{% if broadcast_message.areas %}
  <div>
    <p class="govuk-!-margin-right-2 area-list">Area: </p>
    <ul class="area-list">
      {% for area in broadcast_message.areas %}
      {% endfor %}
    </ul>
  </div>
{% endif %}
```